src/WhereConditions/WhereConditionsServiceProvider.php

```diff
@@ -43,14 +44,21 @@ function (RegisterDirectiveNamespaces $registerDirectiveNamespaces): string {
         $dispatcher->listen(
             ManipulateAST::class,
             function (ManipulateAST $manipulateAST): void {
-                /** @var \Nuwave\Lighthouse\WhereConstraints\Operator $operator */
+                /** @var \Nuwave\Lighthouse\WhereConditions\Operator $operator */
                 $operator = $this->app->make(Operator::class);
 
                 $manipulateAST->documentAST
                     ->setTypeDefinition(
-                        static::createWhereConstraintsInputType(
-                            static::DEFAULT_WHERE_CONSTRAINTS,
-                            'Dynamic WHERE constraints for queries.',
+                        static::createWhereConditionsInputType(
+                            static::DEFAULT_WHERE_CONDITIONS,
+                            'Dynamic WHERE conditions for queries.',
+                            'String'
+                        )
+                    )
+                    ->setTypeDefinition(
+                        static::createWhereConditionsInputType(
+                            static::DEFAULT_WHERE_HAS_CONDITIONS,
```

Comment:
I don't think we need that second type, since they are the same. Makes reusage on the client easier when using something like TypeScript.

Reply:
Done!